spec/models/publish_services_event_metadata_spec.rb

```diff
@@ -340,7 +340,7 @@
             .times(3)
 
           expect(Rails.logger).to receive(:error)
-            .with("Error getting signing certificates for entity_id: #{new_msa_encryption_certificate.component.entity_id}! (Code: 404)")
+            .with("Error getting signing certificates for entity_id: #{msa_signing_certificate.component.entity_id}! (Code: 404)")
```

Comment:
Why has this changed?

Reply:
So there are two identical set of tests, one for encryption, one for signing. I assume it was copy and pasted over and someone didnt change it, so it has always been checking the wrong thing.